packages/runtime/test/dataViews/requestItems/ShareAttributeRequestItemDVO.test.ts

```diff
@@ -224,7 +223,6 @@ describe("ShareAttributeRequestItemDVO", () => {
             query: { "content.value.@type": "DisplayName", "shareInfo.peer": rAddress }
         });
         const senderMessage = await exchangeAndAcceptRequestByMessage(sRuntimeServices, rRuntimeServices, requestContent, responseItems);
-        await sEventBus.waitForEvent(OutgoingRequestStatusChangedEvent, (e) => e.data.newStatus === LocalRequestStatus.Completed);
         const dto = senderMessage;
```

Comment:
shouldn't this be re-added? same for other files where events were removed

Reply:
this is included in the method `exchangeAndAcceptRequestByMessage` above, I was wrong to put this line in in the first place

Reply:
IC